base_layer/core/src/transactions/transaction.rs

```diff
@@ -640,6 +641,19 @@ impl Transaction {
     }
 }
 
+impl Add for Transaction {
```

Comment:
Nice "Add" :)

Is it wrong to run cut-through before returning in this impl?

Reply:
On a functional level, probably not, but I think it might be confusing if you dont know it happens in the background and your inputs and outputs are disappearing.